### ISSUE TITLE
Hold connection open when device is locked

### ISSUE THREAD

**Issue by bhr** [Lejdborg/NMSSH#79]
Hi,

I'm not quite sure whether this is a feature or a bug.

When testing on device, I experienced that the connection will be closed when the device is locked.

Do you know a way to let the connection stay open or is a manual reconnect my only option?

Thank you.

Cheers

**Comment by crazyquark** [comment]
Yup, I see this too. So - we need to disconnect manually the session and reconnect?

**Comment by pomfort** [comment]
AFAIK, yes. That's what I'm doing.

**Comment by bhr** [comment]
AFAIK, yes. That's what I'm doing.
